.github/workflows/release.yaml

```diff
@@ -0,0 +1,54 @@
+name: Gingersnap cache-manager - Build and publish
+
+on: 
+  push:
+    tags:
+      - 'v*'
+      
+env:
+  GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}    
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get latest tag
+        id: vars
+        run: | 
+          LATEST_TAG=$(git describe --tags `git rev-list --tags --max-count=1`)
+          echo LATEST_TAG=$LATEST_TAG >> $GITHUB_OUTPUT
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Build
+        run: mvn package -DskipTests
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Quay
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: manager/src/main/docker/Dockerfile.jvm
+          push: true
+          tags: quay.io/gingersnap/cache-manager:${{ steps.vars.outputs.LATEST_TAG }}
+          build-args: |
+            cache_manager_VERSION=${{ steps.vars.outputs.LATEST_TAG }}
```

Comment:
We can remove this build-arg as it's never used by the Dockerfile.

Reply:
Removed. thanks